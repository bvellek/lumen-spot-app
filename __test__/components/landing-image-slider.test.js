import React from 'react';
import renderer from 'react-test-renderer';

import LandingImageSlider from '../../public/js/components/landing-image-slider';

describe('<LandingImageSlider />', () => {
  it('it renders correctly', () => {
    const component = renderer.create(
      <LandingImageSlider />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
