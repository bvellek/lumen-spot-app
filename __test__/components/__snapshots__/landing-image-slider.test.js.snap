// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingImageSlider /> it renders correctly 1`] = `
<section
  className="photo-slide-section"
>
  <div
    id="slider"
  >
    <figure>
      <img
        alt="circle on far left"
        src="./img/img-slider/01-circle.jpg"
      />
      <img
        alt="circle moved toward right"
        src="./img/img-slider/02-circle.jpg"
      />
      <img
        alt="circle centered"
        src="./img/img-slider/03-circle.jpg"
      />
      <img
        alt="circle to right"
        src="./img/img-slider/04-circle.jpg"
      />
      <img
        alt="circle on far left"
        src="./img/img-slider/01-circle.jpg"
      />
    </figure>
  </div>
</section>
`;
